components/user_limit_modal/user_limit_modal.test.tsx

```diff
@@ -8,6 +8,8 @@ import UserLimitModal from './user_limit_modal';
 
 describe('component/UserLimitModal', () => {
     const requiredProps = {
+        userIsAdmin: false,
```

Comment:
Could we add a test here that the correct button/link is shown in this modal if admin or not?